tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/calendar/CalendarEventReplyGeneratorTest.scala

```diff
@@ -0,0 +1,220 @@
+package com.linagora.tmail.james.jmap.calendar
+
+import java.io.ByteArrayInputStream
+import java.util.stream.Stream
+
+import com.linagora.tmail.james.jmap.model.{AttendeeReply, CalendarEventParsed, CalendarEventReplyGenerator => testee}
+import net.fortuna.ical4j.model.Calendar
+import net.fortuna.ical4j.model.component.{VEvent, VTimeZone}
+import net.fortuna.ical4j.model.parameter.{Cn, PartStat}
+import net.fortuna.ical4j.model.property.Attendee
+import org.apache.james.core.Username
+import org.assertj.core.api.Assertions.{assertThat, assertThatCode, assertThatThrownBy}
+import org.assertj.core.api.SoftAssertions.assertSoftly
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.{Arguments, MethodSource}
+
+import scala.jdk.CollectionConverters._
+
+object CalendarEventReplyGeneratorTest {
+  val partStats: Stream[Arguments] = {
+    Stream.of(
+      Arguments.of(PartStat.ACCEPTED),
+      Arguments.of(PartStat.DECLINED),
+      Arguments.of(PartStat.TENTATIVE))
+  }
+}
+
+class CalendarEventReplyGeneratorTest {
```

Comment:
Edge case idea: what happen when I'm not attending the event (not listed as an attendee)?

Reply:
I just tested on GMAIL
A send invite to B
B forwarded invite email to C
C click YES
-> A received accepted email from C

Reply:
Checked the `ics` file
The request ics file: NO ATTENDEE C
The reply ics file: Remove B, and replace by C

Reply:
I don't think this behavior is correct then? If C was not invited initially, he should not be able to do an action on it even if he got it forwarded by the initial attendee?

Reply:
I think so, the view from the user case is incorrect.

Reply:
Argee on that too.

Reply:
updated